docker/addons/mongodb-reader/docker-compose.yml

```diff
@@ -25,8 +25,12 @@ services:
       MF_MONGO_READER_DB_NAME: ${MF_MONGO_READER_DB_NAME}
       MF_MONGO_READER_DB_HOST: mongodb
       MF_MONGO_READER_DB_PORT: ${MF_MONGO_READER_DB_PORT}
+      MF_MONGO_READER_SERVER_CERT: ${MF_MONGO_READER_SERVER_CERT}
```

Comment:
are these variables in .env file ?

Reply:
I forgot to add them, they are in .env now. Thanks!